lib/user_repository.py

```diff
@@ -0,0 +1,27 @@
+from lib.user import User
+
+
+class UserRepository:
+    def __init__(self, connection):
+        self.connection = connection  
+
+    def all(self):
+        rows = self.connection.execute('SELECT * from users')
+        users = []
+        for row in rows:
+            item = User(row["id"], row["username"], row["email"], row['password'])
+            users.append(item)
+        return users
+
+    def create(self, user):
+        rows = self.connection.execute('INSERT INTO users (username, email, password) VALUES (%s, %s, %s) RETURNING id', [
```

Comment:
Ooh, I've not come across 'RETURNING id' before! Could we talk about it at the next meeting?